planner/core/common_plans.go

```diff
@@ -192,6 +193,7 @@ func (e *Execute) getPhysicalPlan(ctx sessionctx.Context, is infoschema.InfoSche
 			if err != nil {
 				return nil, errors.Trace(err)
 			}
+			log.Errorf("hit: %+v", ToString(plan))
```

Comment:
Why here use the level of “Error”？

Reply:
Thank you for the comment. The PR should remove the debugging code. I'll delete the line from the updated PR.